src/EcommerceStatsBundle/Bridge/Chameleon/Twig/FormatterTwigExtension.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
```

Comment:
license?

Reply:
I must have missed this class.

:heavy_check_mark: Added license header. Doublechecked all other classes, too